.github/workflows/submit.yml

```diff
@@ -185,7 +185,9 @@ jobs:
           path: gtest
 
       - name: Install dependencies
-        run: sudo apt-get install libxrandr-dev libxtst-dev libcups2-dev libasound2-dev
+        run: |
+          sudo apt-get install libxrandr-dev libxtst-dev libcups2-dev libasound2-dev
+          sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 100 --slave /usr/bin/g++ g++ /usr/bin/g++-10
```

Comment:
Maybe we should use apt-get functionality to install a specific version of packages? I'm not sure how relevant it is for the X and alsa libraries since they change very seldom, but perhaps for gcc, to get a specific point release of the compiler.

Reply:
Something along the lines of `apt-get install gcc-10=10.2.0-5ubuntu1~20`

Reply:
That sounds reasonable, I don't know how often these change in Ubuntu LTS, but can't hurt to be explicit here as well.